openprescribing/frontend/tests/commands/test_import_measures.py

```diff
@@ -31,15 +36,24 @@ def isclose(a, b, rel_tol=0.001, abs_tol=0.0):
         return a == b
 
 
-def test_measures():
+def _get_measure_fixture(name):
     fpath = settings.REPO_ROOT
     fname = os.path.join(
-        fpath, ('openprescribing/frontend/tests/fixtures/'
-                'cerazette_measure.json'))
-    json_data = json.load(open(fname))
-    return {
-        'cerazette': json_data
-    }
+        fpath, ("openprescribing/frontend/tests/fixtures/measure_definitions/"
+                "{}.json".format(name)))
```

Comment:
This is, to me, really oddly formatted.  I would much rather see us break the line limit here, than do this.